test/colony.js

```diff
@@ -827,6 +855,32 @@ contract("Colony", addresses => {
       assert.equal(potBalance.toNumber(), 250);
     });
 
+    it("should disburse nothing for unsatisfactory work", async () => {
```

Comment:
Strictly, this only checks we disburse nothing for unsatisfactory work from the worker and the manager. We check that the evaluator failing to rate assigns them as a '1' in another test, but I don't see anywhere we test that the evaluator doesn't get their payout?

Reply:
Elsewhere we test that the evaluator receives a 1 rating, and so we've "transitively" tested that they can't get payout (since payout is based on the rating field and all roles have the same struct). I can add an explicit end-to-end test.

Reply:
We test the manager and work explicitly in this test. Just seems odd to leave the evaluator out of the fun.

Reply:
Fair enough.

Reply:
I just note that it'll take a second test -- if evaluator receives a 1 (did not rate) then the worker automatically receives a 3.